pages/knowledge/colours-and-contrast/user-interface-components/README.md

```diff
@@ -0,0 +1,52 @@
+---
+navigation_title: "Colour contrast for UI components"
+position: 4
+changed: "2019-04-21"
+---
+
+# Colour contrast for User Interface components
+
+**“User Interface (UI) components” are interactive elements, ranging from simple buttons and input fields to complex UI widgets. Just as for text there is a general minimal contrast level that needs to be reached to make their functionality and content accessible.**
+
+## Minimal contrast ratio
+
+Version 2.1 of the [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#non-text-contrast) of `3:1` against adjacent colour(s) for all User Interface (UI) components.
+
+## Example: form controls
```

Comment:
I'm unsure about the wording "Example". We use "example" pretty strictly for our code examples, and we do not have any of these on this page. Could maybe "dilute" (verwässern) the strict meaning a bit?

Reply:
Leave it for now